dagstore_test.go

```diff
@@ -583,75 +583,6 @@ func TestLazyInitialization(t *testing.T) {
 	require.EqualValues(t, 16, info.refs)
 }
 
-// TestThrottleFetch exercises and tests the fetch concurrency limitation.
-// Testing thottling on indexing is way harder...
-func TestThrottleFetch(t *testing.T) {
```

Comment:
Thanks for the unit test at the upgrader level, but it would be great to keep the integration test too.